src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -63,7 +67,7 @@
      * @return JSONArray
      * @throws IOException
      */
-    public JSONArray filterFormParam(JSONObject criteriaObj, JSONObject optionsObj) {
+    public JSONArray filterFormParam(JSONObject criteriaObj, JSONObject optionsObj, String filterKey) {
         JSONArray resultAggregatedObject;
         String criteria = editObjectNameInQueryParam(criteriaObj);
         
```

Comment:
And Options and filter should be optional and not mandatory, I guess. It should be possible to use the query with only"criteria".

Reply:
Options and filter are actually optional. And it is possible to only use criteria (criteria is not optional).